### PR TITLE
Tickets/dm 33851

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: doc/user-guide/user-guide.rst [hunk 1]
```diff
@@ -111,21 +111,22 @@ To deal with this process in a more comprehensive way, this repository uses a di
...
-      By default GitHub will assign the ``master`` branch as a target to all PRs.
+      By default GitHub will assign the ``main`` branch as a target to all PRs.
       You must manually select the cycle branch in this occasion.
 
-  - If new revisions to the current cycle (say cycle 20) are needed to support the production environment, those changes must be done on ticket branches and merged back to ``master``, as usual.
+  - If new revisions to the current cycle (say cycle 20) are needed to support the production environment, ticket branches out of ``main`` should be created and PRs must be opened and merged back to ``main``, as usual.
 
-  - If new revisions to the new cycle are needed, before the new cycle is officially released, ticket branches out of the new cycle branch should be created (e.g. ``cycle/0021``) and PRs must be opened and merged **back to the cycle branch**.
+  - If new revisions to the new cycle are needed **before the new cycle is officially released**, ticket branches out of the new cycle branch should be created (e.g. ``cycle/0021``) and PRs must be opened and merged **back to the cycle branch**.
```
Comment (tribeiro) [inline doc/user-guide/user-guide.rst:119]: I think we need to add something saying that the revision should not be increased while the new revision is not officially released.

**Review by tribeiro — APPROVED** [review]
I made a few comments inline, please take a look before merging...

File: doc/user-guide/user-guide.rst [hunk 2]
```diff
@@ -94,11 +94,11 @@ To deal with this process in a more comprehensive way, this repository uses a di
 
 .. _TSSW: https://tssw-developer.lsst.io/#id4
 
-  - For the cycle build we adopt ``master`` as the main branch, and also the branch that tracks what is running in the production environment.
+  - For the cycle build we adopt ``main`` as the main branch, and also the branch that tracks what is running in the production environment.
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply doc/user-guide/user-guide.rst:97]: Done

File: doc/user-guide/user-guide.rst (hunk 1 above)
Comment (wvreeven) [reply doc/user-guide/user-guide.rst:119]: Good point. I added another bullet explaining that.

**Pull request merged**